panic: runtime error: invalid memory address or nil pointer dereference

Error after syncing for around 15 minutes:

```
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18197/196041 - estimated time remaining 1h47m58s blocksPerSecond=27.4 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18198/196041 - estimated time remaining 1h47m47s blocksPerSecond=27.5 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18199/196041 - estimated time remaining 1h47m35s blocksPerSecond=27.6 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18200/196041 - estimated time remaining 1h47m23s blocksPerSecond=27.6 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18202/196041 - estimated time remaining 1h47m11s blocksPerSecond=27.6 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18205/196041 - estimated time remaining 1h47m0s blocksPerSecond=27.7 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18206/196041 - estimated time remaining 1h46m48s blocksPerSecond=27.8 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18207/196041 - estimated time remaining 1h46m36s blocksPerSecond=27.8 peers=10/25
[2019-12-08 21:56:14]  INFO initial-sync: Processing block 18208/196041 - estimated time remaining 1h46m25s blocksPerSecond=27.9 peers=10/25
[2019-12-08 21:56:16]  INFO p2p: New peer connected peer=16Uiu2HAmUYj89iS7jCXrnrGcrvdDgfBXB1H5zUis1GJXS7ovn467
[2019-12-08 21:56:16]  INFO p2p: New peer connected peer=16Uiu2HAmUVvgkZMYj2qriHeYP4FwjwdG87479eYdhdjy2pTAdnbY
[2019-12-08 21:56:23]  INFO p2p: New peer connected peer=16Uiu2HAmUYj89iS7jCXrnrGcrvdDgfBXB1H5zUis1GJXS7ovn467
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x11a2a45]

goroutine 945916 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.bestFinalized(0x1479f20, 0x0, 0xc001e7f680, 0x43e0d6, 0xc00515ff80, 0x70, 0x70)
	beacon-chain/sync/initial-sync/round_robin.go:309 +0x145
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.highestFinalizedEpoch(...)
	beacon-chain/sync/initial-sync/round_robin.go:295
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).roundRobinSync.func2(0x452e, 0x48, 0x1, 0xc00ad49a00, 0x7, 0x10, 0x1, 0x5, 0x0, 0x1a7e560, ...)
	beacon-chain/sync/initial-sync/round_robin.go:86 +0x24b
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).roundRobinSync.func2.1(0xc00700a130, 0xc00a7fb980, 0xc00041a5a0, 0x1aad780, 0xc002b6bbc0, 0xc0068fa640, 0xc00ad49a00, 0x8, 0x10, 0xc00a7fb8c0, ...)
	beacon-chain/sync/initial-sync/round_robin.go:139 +0x9eb
created by github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*InitialSync).roundRobinSync.func2
	beacon-chain/sync/initial-sync/round_robin.go:115 +0x530

```

--- a/beacon-chain/sync/initial-sync/round_robin.go
+++ b/beacon-chain/sync/initial-sync/round_robin.go
@@ -306,6 +306,9 @@ func bestFinalized() ([]byte, uint64, []peer.ID) {
 	rootToEpoch := make(map[[32]byte]uint64)
 	for _, k := range peerstatus.Keys() {
 		s := peerstatus.Get(k)
+		if s == nil {
+			continue
+		}
 		r := bytesutil.ToBytes32(s.FinalizedRoot)
 		finalized[r]++
 		rootToEpoch[r] = s.FinalizedEpoch

